modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -437,7 +437,8 @@ image::osv2-ui.png[]
 you can delete a partition or the store itself by hovering and clicking the trash
 can icon.
 
-*Note:* Object Store v2 provides persistent storage of objects with a time to live (TTL) of up to 30 days.
+[NOTE]
+Object Store v2 provides persistent storage of objects with a time to live (TTL) of up to 30 days.
```

Comment:
ditto prev. question about this sentence.

Reply:
Changed to: Objects are stored for up to 30 days, based on the TTL (time to live).